app/services/document_url.rb

```diff
@@ -37,7 +37,11 @@ def auth_bypass_id
   attr_reader :document
 
   def secret_token_for_preview_url
-    JWT.encode({ 'sub' => auth_bypass_id }, ENV.fetch('JWT_AUTH_SECRET'), 'HS256')
+    JWT.encode(
```

Comment:
Hanging parenthesis have been a long running part of our style guide: https://github.com/alphagov/styleguides/blob/7a9045bf653e6f12d614217b30d70b7798ece5bd/ruby.md#general and thus is the prevailing gov.uk convention.

They really should be in the linter to save this being discussions since the above doc is now replaced with a link to the linter 🤕

Reply:
Ah, if it's in some sort of style guide, that's totally fine and I'll try to do similar.